flang/lib/Lower/ConvertVariable.cpp

```diff
@@ -956,7 +956,16 @@ static void instantiateLocal(Fortran::lower::AbstractConverter &converter,
                              Fortran::lower::SymMap &symMap) {
   assert(!var.isAlias());
   Fortran::lower::StatementContext stmtCtx;
+  // isUnusedEntryDummy must be computed before mapSymbolAttributes.
+  const bool isUnusedEntryDummy =
+      var.hasSymbol() ? Fortran::semantics::IsDummy(var.getSymbol()) &&
+                            !symMap.lookupSymbol(var.getSymbol()).getAddr()
+                      : false;
```

Comment:
Did you consider
```
  const bool isUnusedEntryDummy =
      var.hasSymbol() && Fortran::semantics::IsDummy(var.getSymbol()) &&
                            !symMap.lookupSymbol(var.getSymbol()).getAddr();
```

And this might be a reasonable time to delete the bogus blank line near the end of this function.

Reply:
Thanks!